applications/ParticleMechanicsApplication/custom_conditions/particle_based_conditions/mpm_particle_penalty_dirichlet_condition.h

```diff
@@ -213,6 +204,8 @@ class MPMParticlePenaltyDirichletCondition
     ///@name Member Variables
     ///@{
 
+    double m_penalty = 1.0;
```

Comment:
I think for the default value 0.0 would be good, as it is the same in "apply_mpm_particle_dirichlet_condition_process.py"

Reply:
I agree, however, it is anyways an error if the default config is used.

Reply:
I think zero is better, more self explanatory

Reply:
still, it is an wrong if it is not applied.